yogstation/code/modules/vending/clothesmate.dm

```diff
@@ -1,94 +0,0 @@
-/obj/machinery/vending/clothing

```

Comment:
Don't delete files just put a comment saying it's mirrored or something

Reply:
oh wait thats a yogs file

Reply:
monster pls